tests/e2e-integration/e2e/sections/home/Home.spec.ts

```diff
@@ -42,6 +42,19 @@ describe('Home Page', () => {
     cy.findByText(/Dataverse Admin/i).should('exist')
   })
 
+  it('finds the "Add Data" button', () => {
+    cy.visit('/spa')
+
+    const addDataBtnNav = cy.get('nav').findByRole('button', { name: /Add Data/i })
+    addDataBtnNav.should('exist')
+    addDataBtnNav.click()
+    cy.get('nav').findByText('New Dataverse').should('be.visible')
+    cy.get('nav').findByText('New Dataset').should('be.visible')
+    cy.get('main')
+      .findByText(/Add Data/i)
+      .should('exist')
+  })
```

Comment:
I believe that since this is an e2e test, what we aim to verify is whether the user is redirected to the corresponding pages. Therefore, I would suggest adding some assertions to check that when the user clicks "New Dataset," they are indeed redirected to the "Create Dataset" form